gluster/file/README.md

```diff
@@ -66,6 +68,13 @@ parameters:
 
 * `gidMin` + `gidMax` : The minimum and maximum value of GID range for the storage class. A unique value (GID) in this range ( gidMin-gidMax ) will be used for dynamically provisioned volumes. These are optional values. If not specified, the volume will be provisioned with a value between 2000-2147483647 which are defaults for gidMin and gidMax respectively.
 
+* `clusterid`: It is the ID of the cluster which will be used by Heketi when provisioning the volume. It can also be a list of comma separated cluster IDs. This is an optional parameter.
```

Comment:
if it can be a list, would `clusterids` still make sense?

Reply:
@rootfs true, however intree is with `clusterid` which cant be changed , so thought of keeping the same for external one.